Maple2.Model/Game/Item/ItemBlueprint.cs

```diff
@@ -1,34 +1,45 @@
-﻿using Maple2.PacketLib.Tools;
+﻿using Maple2.Model.Enum;
+using Maple2.PacketLib.Tools;
 using Maple2.Tools;
 
 namespace Maple2.Model.Game;
 
 public sealed class ItemBlueprint : IByteSerializable, IByteDeserializable {
+    public long BlueprintUid;
```

Comment:
**suggestion (performance):** Evaluate the necessity of all new ItemBlueprint properties

The ItemBlueprint class has been expanded with several new properties. Review if all of these are necessary and consider if any could be combined or represented more efficiently to reduce memory usage.